src/main/java/com/dynatrace/openkit/protocol/StatusResponse.java

```diff
@@ -24,13 +24,13 @@
 public class StatusResponse extends Response {
 
     // status response constants
-    private static final String RESPONSE_KEY_CAPTURE = "cp";
-    private static final String RESPONSE_KEY_SEND_INTERVAL = "si";
-    private static final String RESPONSE_KEY_MONITOR_NAME = "bn";
-    private static final String RESPONSE_KEY_SERVER_ID = "id";
-    private static final String RESPONSE_KEY_MAX_BEACON_SIZE = "bl";
-    private static final String RESPONSE_KEY_CAPTURE_ERRORS = "er";
-    private static final String RESPONSE_KEY_CAPTURE_CRASHES = "cr";
+    public static final String RESPONSE_KEY_CAPTURE = "cp";
```

Comment:
change to pkg default

fix also in TimeSyncResponse

Reply:
In TimeSyncResponse it's not possible, since there are other test using the constants (different package)